### DISCUSSION TITLE
How to set IconButton label value in custom plugin?

### DISCUSSION THREAD

**Discussion by ammoyf** [igordanchenko/yet-another-react-lightbox#302] [Q&A] [answered]
Starting from 3.2x.xx, I noticed types for label are added.

[3d12ec7](https://github.com/igordanchenko/yet-another-react-lightbox/commit/3d12ec753caa7d52db69745a9bab8df733dca21b)

In my custom plug in, how can I set the custom label value. Currently only the keys of type Labels are accepted. Can you kindly let me know how can I infer my label type?

<img width="691" alt="Screenshot 2024-08-14 at 11 29 13" src="https://github.com/user-attachments/assets/91a4db25-087b-4547-9de7-d5c0ccec1617">

**Comment by igordanchenko** [comment, marked as answer]
You can register custom labels with the following module augmentation:

```tsx
declare module "yet-another-react-lightbox" {
  interface Labels {
    Rotate?: string;
  }
}
```

Thank you for pointing this out, I'll update the docs.
